testing/src/test/java/io/stargate/it/http/docsapi/DocumentApiV2Test.java

```diff
@@ -21,30 +18,30 @@
 import io.stargate.web.models.Keyspace;
 import io.stargate.web.models.ResponseWrapper;
 import java.io.IOException;
-import java.net.InetSocketAddress;
+import java.lang.reflect.Method;
 import java.net.URLEncoder;
-import java.time.Duration;
 import java.util.HashSet;
 import java.util.Iterator;
 import java.util.List;
+import java.util.Optional;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
-import net.jcip.annotations.NotThreadSafe;
 import okhttp3.MediaType;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
 import okhttp3.RequestBody;
 import okhttp3.Response;
 import okhttp3.ResponseBody;
 import org.apache.http.HttpStatus;
-import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.parallel.Isolated;
 
-@NotThreadSafe
+@Isolated
```

Comment:
We need to rework this test since it seems like there's a lot of interdependency between methods.